.github/workflows/builds.yml

```diff
@@ -224,6 +244,28 @@ jobs:
           name: physmon
           path: physmon
 
+      - name: ActsAnalysisResidualsAndPulls checks
+        shell: bash
+        run: >
+          echo "::group::Dependencies"
+          && pip3 install histcmp==0.4.1
+          && test -d /usr/local/share/Geant4-10.6.1/data || /usr/local/bin/download_geant4_data.sh
+          && source /usr/local/bin/thisroot.sh
+          && source /usr/local/bin/thisdd4hep_only.sh
+          && source /usr/local/bin/geant4.sh
+          && source build/python/setup.sh
+          && echo "::endgroup::"
+          && export PYTHONPATH="${PYTHONPATH}":"${GITHUB_WORKSPACE}/Examples/Scripts/Python"
+          && thirdparty/OpenDataDetector/ci/full_chain_odd.py
+          && build/bin/ActsAnalysisResidualsAndPulls -i trackstates_ckf.root -o acts_analysis_residuals_and_pulls.root --predicted --filtered --smoothed --silent || true
+          && histcmp acts_analysis_residuals_and_pulls.root CI/physmon/reference/acts_analysis_residuals_and_pulls.root
```

Comment:
`histcmp -o report.html` should produce the output HTML report, and `-p $folder` will dump all the comparison plots as PDFs in `$folder`. Both of these ideally would go into the folder that get's attached as an artifact. 

Can you check how large this gets? I think with the large number of plots that we get out of the root files, this might be a bit too much.

Reply:
looks like it is getting really big https://github.com/andiwand/acts/actions/runs/2293280651

are there any options to filter what we want to compare / plot?